tests/conftest.py

```diff
@@ -16,18 +16,60 @@
 """
 import pytest
 import os
-
+import numpy as np
+import pennylane as qml
+from pennylane.plugins import DefaultGaussian
 
 # defaults
 TOL = 1e-3
 
+class DummyDevice(DefaultGaussian):
+    """Dummy device to allow Kerr operations"""
+    _operation_map = DefaultGaussian._operation_map.copy()
+    _operation_map['Kerr'] = lambda *x, **y: np.identity(2)
+
 
 @pytest.fixture(scope="session")
 def tol():
     """Numerical tolerance for equality tests."""
     return float(os.environ.get("TOL", TOL))
 
 
+@pytest.fixture(scope="session")
```

Comment:
Double check which ones of these are still used, e.g., I believe `tol_sampl` is no longer needed(?)